is_prime.py

```diff
@@ -0,0 +1,17 @@
+import math
+def is_prime(max_number):
+# prime factors
```

Comment:
If it is a shared module, please add documentation to this function. 
Write what is it doing, what parameters it takes, and what is returning.